operators/extraordinary_patch_constructor.py

```diff
@@ -57,6 +57,23 @@ def get_neighbor_verts(cls, vert) -> list:
         nb_verts.append(vert)
 
         return nb_verts
+    @classmethod
+    def get_bezier_patch(cls, vert) -> list:
```

Comment:
Could you reduce the duplicated code of "get_bezier_patch" and "get_patch"?
e.g. get_patch calls get_bezier_patch